extensions/vertx-http/deployment/src/main/java/io/quarkus/devui/deployment/DevUIProcessor.java

```diff
@@ -805,19 +810,19 @@ private List<String> listOrString(Map<String, Object> metaData, String key) {
         return List.of(String.valueOf(value));
     }
 
-    private void produceResources(String artifactId,
+    private void produceResources(ResolvedDependency runtimeExt,
             BuildProducer<WebJarBuildItem> webJarBuildProducer,
             BuildProducer<DevUIWebJarBuildItem> devUIWebJarProducer) {
 
-        GACT gact = getGACT(artifactId);
-        String namespace = getNamespace(gact);
+        String namespace = getNamespace(runtimeExt.getKey());
         if (namespace.isEmpty()) {
             namespace = "devui";
         }
         String buildTimeDataImport = namespace + "-data";
 
+        final GACT deploymentKey = getDeploymentKey(runtimeExt);
```

Comment:
@phillip-kruger I just wanted to make sure we really need the deployment key here. The previous code was simply appending `-deployment` to the runtime artifactId.